task4/func.js

```diff
@@ -0,0 +1,21 @@
+function Person(name,age) {
+    this.name = name;
+    this.age = age;
+}
+
+Person.prototype.introduce = function () {
+    return 'My name is ' + this.name + ' and I am ' + this.age;
```

Comment:
I know it's from the sample, but let's make it beautiful with template string :)
`My name is ${this.name} and I am ${this.age}`